layouts/basic/modules/Settings/Currency/ListViewContents.tpl

```diff
@@ -64,6 +64,8 @@
 							{else}
 								&nbsp;{\App\Language::translate('LBL_INACTIVE',$QUALIFIED_MODULE)}
 							{/if}
+						{elseif $LISTVIEW_HEADERNAME eq 'currency_name'}
+							{\App\Language::translate($LISTVIEW_ENTRY->getDisplayValue('currency_code'), 'Other.Currency')}
```

Comment:
add getDisplayValue function  in modules/Settings/Currency/models/Record.php
similar to
https://github.com/YetiForceCompany/YetiForceCRM/blob/f8e0e1e6ada1d8a48d642167d0a90fedcf41c297/modules/Settings/Companies/models/Record.php#L90-L124